scripts.js

```diff
@@ -7,61 +7,65 @@ let java = document.getElementById('java');
 let counterang = 0, counterhtml = 0, countercss = 0, counterjs = 0, counterc = 0, counterjava = 0;
 
 setInterval(() => {
-  if(counterang === 5){
+  if (counterang === 5) {
     clearInterval;
   } else {
     counterang++;
     angular.innerHTML = `${counterang}%`;
   }
-  
+
 }, 100)
 
 setInterval(() => {
-  if(counterhtml === 30){
+  if (counterhtml === 30) {
     clearInterval;
   } else {
     counterhtml++;
     html.innerHTML = `${counterhtml}%`;
   }
-  
+
 }, 65)
 
 setInterval(() => {
-  if(countercss === 30){
+  if (countercss === 30) {
     clearInterval;
   } else {
     countercss++;
     css.innerHTML = `${countercss}%`;
   }
-  
+
 }, 65)
 
 setInterval(() => {
-  if(counterjs === 10){
+  if (counterjs === 10) {
     clearInterval;
   } else {
     counterjs++;
     js.innerHTML = `${counterjs}%`;
   }
-  
+
 }, 180)
 
 setInterval(() => {
-  if(counterc === 50){
+  if (counterc === 50) {
     clearInterval;
   } else {
     counterc++;
     c.innerHTML = `${counterc}%`;
   }
-  
+
 }, 40)
 
 setInterval(() => {
-  if(counterjava === 25){
+  if (counterjava === 25) {
     clearInterval;
   } else {
     counterjava++;
     java.innerHTML = `${counterjava}%`;
   }
-  
+
 }, 80)
+
+function redirectButton() {
+  location.href = "#start";
+}
```

Comment:
```suggestion
const redirectButton = () => {
  location.href = "#start";
}
```
Use a the modern arrow function.